### DISCUSSION TITLE
Document update Request: Digital Ocean no longer has redash on the marketplace

### DISCUSSION THREAD

**Discussion by ltrainstg** [getredash/redash#6812] [Q&A] [answered]
# Problem 
I was looking at the setup docks and planned to use Digital Ocean and it looks like the marketplace link no longer has redash. 
I know this is open source so did someone just not renew/update the availability or is it intentionally gone? 

[Documentation link ](https://redash.io/help/open-source/setup#do)
[Missing Redash link ](https://marketplace.digitalocean.com/apps/redash)

# Solution
I can see two potential solutions for this request.  
1. Get redash back on the marketplace 
2.  Remove/update the digital ocean setup information. 
 
# Last mention of redash on marketplace 
[Where is the source code when installation is done using DigitalOcean Marketplace #6587
](https://github.com/getredash/redash/discussions/6587)

**Comment by justinclift** [comment]
Interesting.  Yeah, we'll probably need to remove it from the docs in the short term.

Longer term though, it's not clear if the Digital Ocean marketplace is all that useful.  Is it something you've used much with other things? :smile:

**Reply by ltrainstg** [reply]
I've never used the market place before, running [setup.sh ](https://github.com/getredash/setup) worked for me.

**Comment by justinclift** [comment, marked as answer]
@ltrainstg The Digital Ocean bit has been removed from the website ([in this PR](https://github.com/getredash/website/pull/702)), thanks to @lucydodo. :smile:
